package/Dockerfile

```diff
@@ -233,4 +233,6 @@ VOLUME /var/log
 
 ENV ETCD_UNSUPPORTED_ARCH ${ETCD_UNSUPPORTED_ARCH}
 
+RUN echo "export CATTLE_RANCHER_WEBHOOK_VERSION=${CATTLE_RANCHER_WEBHOOK_VERSION}" > /var/lib/rancher/.env
```

Comment:
Relying on an actual file feels weird. Perhaps instead of fighting just this one env var we make a script system that is the bases for some release automation that runs locally pre-build and puts the right versions (of whatever) in the right files through some `sed` magic. What you think @MbolotSuse , we've discussed this before.

Reply:
I've redone the solution.